### ISSUE TITLE
Settings for "Make Similar Images"

### ISSUE THREAD

**Issue by spiralofhope** [easydiffusion/easydiffusion#1335]
- An image has been completed
- Hovering it over gives several links
- One of the links is "Make Similar Images"
- Clicking it will automatically begin a new task with five images
- That new task has settings which do not have configuration options

Inference steps, for example, cannot be adjusted.  In my case, I want 20 because my model works well with 20-30 and my weak graphics card is wasting time making 50 steps.

Given the complexity of the settings tab, maybe this group of settings would also necessitate configuration sub-tabs for clarity?

**Comment by JeLuF** [comment]
Instead of using the "Make Similar Images" shortcut, you can use the "use as input" button. This provides you with control over all the settings.

**Comment by spiralofhope** [comment]
> Instead of using the "Make Similar Images" shortcut, you can use the "use as input" button.

I agree. However, the problem with this is an autosave JSON will embed the input image, which is a huge waste of space for me.  Maybe I should use make similar images but ask for a JSON autosave image embed checkbox?

**Comment by JeLuF** [comment]
"Make similar images" creates an img2img task, just like "Use as input" does. The JSON metadata is the same.

**Comment by fdwr** [comment]
> That new task has settings which do not have configuration options

Yeah, I encountered that too where "Make Similar Images" sadly drops certain useful information in the new task, like the number of images per batch. I frequently generate 6 images with 3 per batch (~3x faster), but then when I click "Make Similar Images" on the existing task, the new task only generates 1 image at a time, which takes 3x longer. So to save generation time, I end up clicking "Use as input" instead and then scrolling back up to the settings to click Generate, and then scrolling back down to the next interesting batch, but scrolling back up and down dozens of times gets tedious in a longer history (which burns time too). It also drops the image count, but the difference between 6 vs 5 images doesn't matter that much :b. If "Make Similar Images" just inherited more of the old task's settings, then it would be an all-around time saver. ⏱

**Comment by JeLuF** [comment]
> 6 images with 3 per batch (~3x faster),

Which system configuration do you have? In all of our benchmarks, running batches in parallel did not have any significant performance improvements (less than 10%).

**Comment by fdwr** [comment]
Hi Jelu. Oops, fixed mistake above (that should have been 30%, not 300%! 😋), but it's faster nonetheless ⏱️. Here are my timings for different batch sizes, 6 per batch, 3 per batch, 1 per batch (all other parameters the same) on my NVidia RTX 2060:

![image](https://github.com/easydiffusion/easydiffusion/assets/1809166/8dd42f7d-c05d-42c1-844d-90154cf7d04b)
